src/client/scripts/views/SignedIn/DiscoverView.js

```diff
@@ -331,6 +309,23 @@ export class DiscoverView {
         if (user.hasHousing && user.description) {
             container.querySelector('.bio-description').innerText = user.description;
         }
+
+        // Adds user social media
+        if(user.socials.ig !== "") {
```

Comment:
Doesn't work, try running

Reply:
That's a bit weird - I'm able to see the uncommented code on my end. Should I take out social media for now?

Reply:
My bad, I see it, it's messed up the layout though
![image](https://github.com/ashleybhandari/team-friendship/assets/99090595/73534d74-4bef-43c0-b68c-fa0161900df3)

Reply:
There's a margin-top for the discover-info class that's really big for some reason

Reply:
That was something I forgot to take out that was used for getting the old filter to render - I'll take it out

Reply:
I committed it - for some reason the error didn't show up on my end when testing initially. Is the layout the same now?

Reply:
It looks good, thank you. Feel free to merge.

Reply:
Sounds good - thanks for your help Ashley!